const promise = Promise.reject('Error!');


